Caddyfile

```diff
@@ -1,31 +1,71 @@
 {
-    order souin_cache before rewrite
-    souin_cache {
+    order cache before rewrite
+    cache {
         headers Content-Type Authorization
         log_level info
+        olric {
+            configuration {
```

Comment:
What's the purpose of this `configuration` level here? Seems useless. Shouldn't it get hoisted up a level?

Reply:
Because we have to separate the configuration from the `path` declaration for the file or the `url`declaration for the cluster address. IMO it's the best compromise.

Reply:
I don't understand - that doesn't seem like a concern that should be exposed to Caddy configuration.

Reply:
We could have some keys conflicts between the whole configuration (placed under `configuration`) and the file configuration (the file location is defined with the `path` one) and the cluster url/ip using the olric client (this is defined with the `url` one).

Reply:
I don't know what you mean because I don't see `path` or `url` in there, I only see `configuration`, which is the only element of `olric`, therefore `configuration` looks redundant.

Reply:
Oh, I'll add other configuration files to be more explicit :)

Reply:
But do we really want to let users configure it via a separate file? That seems counterproductive to me. We should keep this focused.

Reply:
Examples are available.